week-2-practise/level-1/2-validation.js

```diff
@@ -2,7 +2,7 @@
 // TIP: You should write an expression that returns a boolean value
 
 var num = 10;
-var isBigEnough; // ONLY EDIT THIS LINE
+var isBigEnough = true; // ONLY EDIT THIS LINE
```

Comment:
This will work but is cheating :slightly_smiling_face: . It says you should write "an expression" (using `num`). That means something like `num === 5` or `num > 5`.